src/ui/components/Button/index.js

```diff
@@ -6,21 +6,24 @@ import './Button.scss';
 
 export default class Button extends React.Component {
   static propTypes = {
+    appearance: PropTypes.string,
     children: PropTypes.node,
     className: PropTypes.string,
     href: PropTypes.string,
     size: PropTypes.string.isRequired,
   }
 
   static defaultProps = {
+    appearance: 'undefined',
     size: 'normal',
   };
 
   render() {
-    const { children, className, href, size, ...rest } = this.props;
+    const { appearance, children, className, href, size, ...rest } = this.props;
     const props = {
       className: classNames('Button', className, {
         'Button--small': size === 'small',
+        'Button--light' : appearance === 'appearance',
```

Comment:
@mstriemer  so sorry for this, you had to review such mistakes. I feel like laughing at myself.